wiki/Community/Project_Loved/en.md

```diff
@@ -86,6 +85,21 @@ For any questions regarding what goes in each category, contact the [osu! Loved
 
 There are no strict rules governing which maps a captain must nominate for Loved, but generally the nominated maps are very well known and appreciated by the community (as the title "Loved" implies). Sometimes a beatmap will also be preferred if it is significantly older than others or contains beatmaps with unique and high-quality mapping styles.
 
+## Website
+
+All of the submitted beatmaps can be seen at the [Loved website](https://loved.sh). 
```

Comment:
This introductory sentence should be reworded entirely in my opinion.

There isn't any previous reference to "submitted beatmaps". The introduction to this new section should first and foremost introduce the site and then explain how it acts as the main submission hub for users to promote maps towards the Loved status.

Reply:
The website is mentioned in line 11 under "To submit a beatmap for Loved, please visit the [beatmap submission page](https://loved.sh/submit) of the Project Loved website and fill out the form."

Reply:
Despite having a mention there, considering it was mentioned up to almost 100+ lines up and we're adding a new header for the Website alone, the introduction should be much meatier than that. Readers will expect to know that information just by going to its respective section.

Reply:
Aight, makes sense